src/components/Card/Card.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Card from './Card';
+
+describe('Card', () => {
+  const movieId = 1;
+  const movieTitle = 'Test Movie';
+  const moviePosterPath = '/test-poster.jpg';
+  const movieImage = 'https://www.test.com';
+
+  it('renders the Card component', () => {
```

Comment:
Image için error olduğunda bi tane placeholder koyalım demiştik. Onun geliştirmesini yaptıktan sonra onun da testini yazalım mı? Bu çok acil değil. Başka bi PR'da yapabilirsin